pkg/object/httpserver/mux.go

```diff
@@ -269,18 +301,22 @@ func (mi *muxInstance) serveHTTP(stdw http.ResponseWriter, stdr *http.Request) {
 
 		// Write access log.
 		logger.LazyHTTPAccess(func() string {
-			// log format:
-			//
-			// [$startTime]
-			// [$remoteAddr $realIP $method $requestURL $proto $statusCode]
-			// [$contextDuration $readBytes $writeBytes]
-			// [$tags]
-			const logFmt = "[%s] [%s %s %s %s %s %d] [%v rx:%dB tx:%dB] [%s]"
-			return fmt.Sprintf(logFmt,
-				fasttime.Format(startAt, fasttime.RFC3339Milli),
-				stdr.RemoteAddr, req.RealIP(), stdr.Method, stdr.RequestURI,
-				stdr.Proto, metric.StatusCode, metric.Duration, metric.ReqSize,
-				metric.RespSize, ctx.Tags())
+			log := &accessLog{
+				Time:        fasttime.Format(startAt, fasttime.RFC3339Milli),
+				RemoteAddr:  stdr.RemoteAddr,
+				RealIP:      req.RealIP(),
+				Method:      stdr.Method,
+				URI:         stdr.RequestURI,
+				Proto:       stdr.Proto,
+				StatusCode:  metric.StatusCode,
+				Duration:    metric.Duration,
+				ReqSize:     printSize(metric.ReqSize),
+				RespSize:    printSize(metric.RespSize),
```

Comment:
using `fmt.Sprintf` directly is more straightforward and makes the code easier to read.